sky/spot/spot_utils.py

```diff
@@ -286,15 +295,20 @@ def stream_logs_by_id(job_id: int, follow: bool = True) -> str:
             spot_status = spot_state.get_status(job_id)
             if spot_status.is_terminal():
                 break
-            status_display.update(
-                _JOB_WAITING_STATUS_MESSAGE.format(status_str=''))
+            logger.info(
+                f'{colorama.Fore.YELLOW}The job is preempted or cancelled. '
```

Comment:
I tried canceling a `sleep 1000` job and didn't see this message being printed. Saw it when manually terminating the spot VM.

Reply:
Ahh, good point. Since we recently changed the behavior of `sky spot cancel`, this will only happen for preemption.